### ISSUE TITLE
All jobs automatically rescheduled with pilot version 7.3.14

### ISSUE THREAD

**Issue by arrabito** [DIRACGrid/DIRAC#5758]
Dear all,

I've tested DIRAC version 7.3.14 in pilots and I found some misbehaviour preventing to run any job correctly.

All jobs that complete successfully ('Done' status) get automatically rescheduled until the Maximum of allowed trials is reached and in the end they keep stuck in 'Rescheduled' status.

In the pilot outputs,  I get the following error:
```
2021-12-29 08:22:37 UTC None/JobWrapper[18515732] INFO: Job finished successfully with no pending requests
2021-12-29 08:22:37 UTC None/JobWrapper[18515732] INFO: Final job status Done ; Execution Complete
2021-12-29 08:22:37 UTC None/JobWrapper[18515732] INFO: EXECUTION_RESULT[CPU] in sendJobAccounting 7.83 6.36 39583.27 27.56 39867.43
2021-12-29 08:22:38 UTC WorkloadManagement/JobAgent ERROR: Exception in submission
Traceback (most recent call last):
  File "/home/condor/execute/dir_57112/DIRAC_1QbyEipilot/diracos/lib/python3.9/site-packages/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py", line 272, in execute
    result_submitJob = self._submitJob(
  File "/home/condor/execute/dir_57112/DIRAC_1QbyEipilot/diracos/lib/python3.9/site-packages/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py", line 637, in _submitJob
    submission = self.computingElement.submitJob(
  File "/home/condor/execute/dir_57112/DIRAC_1QbyEipilot/diracos/lib/python3.9/site-packages/DIRAC/Resources/Computing/InProcessComputingElement.py", line 92, in submitJob
    os.unlink(inputFile)
TypeError: unlink: path should be string, bytes or os.PathLike, not tuple
2021-12-29 08:22:38 UTC WorkloadManagement/JobAgent WARN: Failure ==> rescheduling (during Job processing failed with exception)
2021-12-29 08:22:38 UTC WorkloadManagement/JobAgent INFO: Job will be rescheduled
2021-12-29 08:22:39 UTC WorkloadManagement/JobAgent ERROR: Failed to reschedule job Maximum number of reschedulings is reached: 3
2021-12-29 08:22:39 UTC WorkloadManagement/JobAgent INFO: JobAgent will stop with message "Problem Rescheduling Job", execution complete.
```
It seems like a bug. I also confirm that reverting back pilot version to 7.3.12, it works fine.

Could you please check?

Thank you.

**Comment by fstagni** [comment]
This, IIRC, was also discovered during the last hackathon and is fixed in https://github.com/DIRACGrid/DIRAC/pull/5745/files but we are missing a new tag/release. 

@atsareg I think we should removed 7.3.14 version.

**Comment by arrabito** [comment]
OK thanks. Good to know.

**Comment by fstagni** [comment]
Fixed in v7.3.15
